bigchaindb_driver/utils.py

```diff
@@ -0,0 +1,41 @@
+from bigchaindb.common.transaction import Asset
+
+
+class CreateOperation:
+    pass
+
+
+class TransferOperation:
+    pass
+
+
+ops_map = {
+    'CREATE': CreateOperation,
+    'TRANSFER': TransferOperation,
+}
+
+
+def _normalize_operation(operation):
+    try:
+        operation = operation.upper()
+    except AttributeError:
+        pass
+
+    try:
+        operation = ops_map[operation]()
+    except KeyError:
+        pass
+
+    return operation
+
+
+def _normalize_asset(asset, is_transfer=False):
+    if is_transfer:
```

Comment:
this routine is a bit unclear

From the input arg `asset` you want to make sure you are returning an `Asset` object
One can pass either a `dict` or a list of `kwargs`?
The switch is based on `is_transfer`? Not on type of the arg `asset`?